docs/docker-compose.yml

```diff
@@ -51,6 +51,12 @@ services:
       SUBLINKS_JWT_SECRET: <JWT-SECRET>
       SUBLINKS_BASE_URL: https://demo.sublinks.org
       SUBLINKS_PICTRS_URL: http://pictrs:8080
+      FEDERATION_QUEUE_ENDPOINT: localhost
```

Comment:
I prefer the nomenclature FEDERATION_QUEUE_HOST rather than FEDERATION_QUEUE_ENDPOINT.